jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/impl/strategy/RoundRobinAssignmentStrategy.java

```diff
@@ -81,7 +81,7 @@ public Assignment apply(Task task, TaskContext taskContext, String excludedUser)
                     Iterator<OrganizationalEntity> groupUsers = userInfo.getMembersForGroup((Group) oe);
                     if (groupUsers != null) {
                         groupUsers.forEachRemaining(user -> {
-                            if (user != null && !excluded.contains(user) && !potentialOwners.contains(user)) {
+                            if (user != null && !user.getId().isEmpty() && !excluded.contains(user) && !potentialOwners.contains(user)) {
```

Comment:
Is this `user.geId()` safe to be non null ?

Reply:
added checking to make it more robust